src/test/isolation2/sql/frozen_insert_crash.sql

```diff
@@ -147,8 +147,7 @@ $$ LANGUAGE plpgsql;
 -- going to be made to disk (we just flushed WALs), so we won't replay it during restart later.
 -- skip FTS probe to prevent unexpected mirror promotion
 1: select gp_inject_fault_infinite('fts_probe', 'skip', dbid) from gp_segment_configuration where role='p' and content=-1;
--- Add panic only for an inserting session.
-1: select gp_inject_fault('qe_exec_finished', 'panic', dbid, sess_id) from gp_segment_configuration join pg_stat_activity on query = 'insert into tab_fi values(2, 2);' where role = 'p' and content = 0;
+1: select gp_inject_fault('qe_exec_finished', 'panic', dbid) from gp_segment_configuration where role = 'p' and content = 0;
```

Comment:
Where did these changes come from in the resulting diff?
It seems you've lost a commit https://github.com/arenadata/gpdb/commit/3d4cc61b6d697e7eb524078285d8dfbb19949e57. You should probably cherry-pick it.

Reply:
Because these changes are not in 8accc02. Ok, let's add it.